### ISSUE TITLE
Add new reference quantized MobileNet model

### ISSUE THREAD

**Issue by psyhtest** [mlperf/inference_policies#48]
Following WG's discussion on 14/Aug/2019, I suggest we document the request and decision here.

@DilipSequeira:
> We would like potentially to provide [an additional] model we trained for MobileNet INT8 to be fine-tuned with the canonical image processing flow, as we think that might recover some of the accuracy lost.”

**Comment by psyhtest** [comment]
Here's a relevant excerpt (quantized MobileNet) from the official README:

| model | framework | accuracy | dataset | model link | model source | precision | notes |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
| mobilenet-v1 quantized | tensorflow | 70.654% | imagenet2012 validation | [from zenodo](https://zenodo.org/record/2269307/files/mobilenet_v1_1.0_224_quant.tgz) | [from tensorflow](http://download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz) | int8 | NHWC |
| mobilenet-v1 quantized | tflite | 70.776% | imagenet2012 validation | [from zenodo](https://zenodo.org/record/2269307/files/mobilenet_v1_1.0_224_quant.tgz) | [from tensorflow](http://download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz) | int8 | NHWC |
| mobilenet-v1 | onnx, pytorch | 71.68% | imagenet2012 validation | [from zenodo](https://zenodo.org/record/3157894/files/mobilenet_v1_1.0_224.onnx) | [from tensorflow](http://download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224.tgz) converted with [this script](https://github.com/mlperf/inference/blob/master/v0.5/classification_and_detection/tools/resnet50-to-onnx.sh) | fp32 | NCHW, tested on pytorch and onnxruntime |

**Comment by christ1ne** [comment]
I suggest we close this as MobileNet is not in the upcoming v0.6 round
